.github/workflows/build.yml

```diff
@@ -81,6 +81,16 @@ jobs:
       - # === Install State Tool ===
         name: Install State Tool
         uses: ActiveState/setup-state-tool@v1
+        if: runner.os != 'Windows'
+
+      - # === Install State Tool (Windows) ===
+        name: Install State Tool (Windows)
+        if: runner.os == 'Windows'
+        shell: pwsh
+        run: |
+          Invoke-Expression -Command ".\installers\install.ps1 -n"
+          echo "$env:LOCALAPPDATA\ActiveState\StateTool\release\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          Invoke-Expression "$env:LOCALAPPDATA\ActiveState\StateTool\release\bin\state config set optin.unstable true"
```

Comment:
This comes from the setup-state-tool GitHub Action and is temporary until this PR's install.ps1 fix makes it to production.